### DISCUSSION TITLE
Not returning distance_scores in novelty sampler

### DISCUSSION THREAD

**Discussion by benwandrew** [AutoResearch/autora#423] [Core Contributions] [answered]
As is, distance_scores are [not returned](https://github.com/AutoResearch/autora-experimentalist-sampler-novelty/blob/539b2e465f0dc6e0c199bf32fde49586be8c259d/src/autora/experimentalist/sampler/novelty/__init__.py#L63) by the `novelty_sampler` function; they should be, correct?

If so, adding to a PR I'm working on.

@musslick @younesStrittmatter

**Comment by musslick** [comment, marked as answer]
Yes that is correct. The novelty sampler has a separate function for that called `novelty_score_sampler`. This is critical because the sampler method itself should only return X so to avoid issues with the experimentalist pipeline.
